flotta-agent.spec

```diff
@@ -7,7 +7,7 @@ Summary:    Agent application for the Flotta Edge Management solution
 ExclusiveArch: %{go_arches}
 Group:      Flotta
 License:    GPL
-Source0:    https://github.com/jakub-dzon/%{name}-%{version}.tar.gz
+Source0:    https://github.com/project-flotta/%{name}-%{version}.tar.gz
```

Comment:
this will not work at all no? Should be flotta-device-worker and there is no target version. Should be `%{name}-%{version}.tar.gz`

Reply:
you're correct. however, do we push it to github? should `Source0` be removed or are we missing some functionality?

Reply:
if you have like that, it'll fail when you do spectool.. so I wouldn't use the URL at all.